extensions/amp-story-panning-media/0.1/amp-story-panning-media.js

```diff
@@ -32,55 +38,110 @@ export class AmpStoryPanningMedia extends AMP.BaseElement {
     /** @private {?Element} */
     this.element_ = element;
 
-    /** @private {?Element} */
-    this.image_ = null;
+    /** @private {?Element} The element that is transitioned. */
+    this.ampImgEl_ = null;
 
-    /** @private {?string} */
-    this.x_ = null;
+    /** @private {?string} Sent to siblings to update their position. */
+    this.x = null;
 
-    /** @private {?string} */
-    this.y_ = null;
+    /** @private {?string} Sent to siblings to update their position. */
+    this.y = null;
 
-    /** @private {?string} */
-    this.zoom_ = null;
+    /** @private {?string} Sent to siblings to update their position. */
+    this.zoom = null;
   }
 
   /** @override */
   buildCallback() {
-    this.x_ = this.element_.getAttribute('x') || '0%';
-    this.y_ = this.element_.getAttribute('y') || '0%';
-    this.zoom_ = this.element_.getAttribute('zoom') || '1';
+    this.ampImgEl_ = dev().assertElement(
+      this.element_.querySelector('amp-img')
+    );
+
+    this.x = this.element_.getAttribute('x') || '0%';
+    this.y = this.element_.getAttribute('y') || '0%';
+    this.zoom = this.element_.getAttribute('zoom') || '1';
+
+    return Services.storyStoreServiceForOrNull(this.win).then(
+      (storeService) => {
+        // If page is active, set panning media state
+        storeService.subscribe(
+          StateProperty.CURRENT_PAGE_ID,
+          (currPageId) => {
+            const isOnActivePage = currPageId === this.getPageId_();
+            if (isOnActivePage) {
+              // TODO(#31932): A key could be sent here to update elements of the same group.
+              // Note, this will not work when there are 2 or more panning components on the same page.
+              // It might need to dynamic to hold more than 1 set of positions.
+              storeService.dispatch(Action.SET_PANNING_MEDIA_STATE, {
+                x: this.x,
+                y: this.y,
+                zoom: this.zoom,
+              });
+            }
+          },
+          true /** callToInitialize */
+        );
+        // If page is active, set panning media state
+        storeService.subscribe(
+          StateProperty.PANNING_MEDIA_STATE,
+          (panningMediaState) => {
+            if (panningMediaState) {
+              const {x, y, zoom} = panningMediaState;
+              // TODO(#31932): Update siblings that are part of the same group.
+              this.updateTransform(x, y, zoom);
```

Comment:
Who calls the `updateTransform` if the active page is the first page? This won't trigger if you don't have the `callToInitialize` parameter